pkg/webui/console/components/webhook-form/index.js

```diff
@@ -225,14 +238,21 @@ export default class WebhookForm extends Component {
             disabled={update}
           />
           <WebhookFormatSelector name="format" required />
+          {reactivate && (
+            <React.Fragment>
+              <div>
+                <Form.SubTitle title={reactivateWebhookTitle} />
+                {reactivateButton}
+              </div>
+            </React.Fragment>
+          )}
```

Comment:
I think we can add this at the top as a `<Notification warning action={…} actionMessage={…}  />`

> This webhook has been deactivated due to several unsuccessful forwarding attemps. It will be automatically reactivated after 24 hours. If you wish to reactivate right away, you can use the "Reactivate"-button.